.eslintrc.json

```diff
@@ -32,7 +32,7 @@
 		"jest/valid-describe": "off",
 		"no-case-declarations": "warn",
 		"react-hooks/rules-of-hooks": "error",
-		"react-hooks/exhaustive-deps": "off",
+		"react-hooks/exhaustive-deps": "error",
```

Comment:
I think we should have this set as `error`, this will serve as safe guard

Reply:
as @manzoorwanijk have noticed few weeks ago we had few `useCallback` deps array empty which causes some issues

Reply:
and I think is important to keep this rule as error since we will start memoizing more aggresively

Reply:
Nice change 👍